pkg/volume/iscsi/iscsi_util.go

```diff
@@ -98,6 +98,7 @@ func (util *ISCSIUtil) MakeGlobalPDName(iscsi iscsiDisk) string {
 }
 
 func (util *ISCSIUtil) AttachDisk(b iscsiDiskBuilder) error {
+	deviceUtil := b.deviceUtil
```

Comment:
why the need for a local variable?  You can reference `b.deviceUtil` throughout this method and not require the local var.